lib/scout_apm/plugs/controller_timer.ex

```diff
@@ -4,10 +4,14 @@ defmodule ScoutApm.Plugs.ControllerTimer do
   def init(default), do: default
 
   def call(conn, _default) do
-    ScoutApm.TrackedRequest.start_layer("Controller", action_name(conn))
+    if !ignore_uri?(conn.request_path) do
```

Comment:
In the ruby agent, we check the ignore after the fact, right as we go to record the request.

This will probably work too, since our recorder checks for a `Controller` layer being present, and skips it if not. 

We can go with this approach, but may want to switch it up later if it gives us any issues.